python/ray/tune/integration/docker.py

```diff
@@ -87,6 +96,7 @@ def sync_up(self, source: str, target: Tuple[str, str]) -> bool:
             target=target_dir,
             down=False,
             ip_address=target_node,
+            should_bootstrap=self._should_bootstrap,
             use_internal_ip=True)
```

Comment:
Should this be set via environment variable instead of an function input? It looks like there would need to be more changes to pass this flag down to here?